symplyphysics/definitions/vector/damping_force_is_proportional_to_velocity.py

```diff
@@ -9,24 +18,76 @@
     validate_output,
 )
 
-# Description
-## Damping force is an external (relative to an object) force that drains energy from the object,
-## reducing the motion of the object. It is a model used, for example, describing the motion
-## of an oscillator.
+damping_constant = Symbol("damping_constant", units.mass / units.time)
+"""
+Non-negative damping constant.
 
-# Law: F = -b * v
-## F - vector of damping force
-## b - damping constant, b >= 0
-## v - vector of object's velocity
+Symbol:
+    :code:`b`
 
-damping_constant = Symbol("damping_constant", units.mass / units.time)
+Dimension: :math:`\mathsf{M} \mathsf{T}^{-1}`
```

Comment:
Remove dimension here to be consistent. We will likely introduce some auto generation of dimensions later.